tools/ansible/tasks/logging/test/logging_test.go

```diff
@@ -37,8 +55,16 @@ func TestPluginsInstalled(t *testing.T) {
 		installedVersion, ok := installedPlugins[pluginName]
 		assert.Truef(t, ok, "Missing plugin %s", pluginName)
 		if pluginVersion != "" {
-			assert.Equalf(t, pluginVersion, installedVersion,
-				"%s expected version: %s, got: %s", pluginName, pluginVersion, installedVersion)
+			// Test version specific to fluentd
+			if pluginName == "fluentd" {
+				// Matching only the version (ex 1.5), not the subversion (ex 1.5.1) of Fluentd because
```

Comment:
Are you sure that sub-version and patch number cannot be pinned?

Reply:
Yes - I tested it again with `gem install fluentd -v '~> 1.5.1'` and it resulted in `fluentd (1.5.2 x64-mingw32)` being installed.

Reply:
Can be fixed with `gem install fluentd -v 1.5.1`. Since this requires changing `install_fluentd` playbook as well, a story created at https://jira.coreos.com/browse/WINC-133.